### ISSUE TITLE
Crash when expanding snippet: "ValueError: zero length field name in format"

### ISSUE THREAD

**Issue by hanschen** [SirVer/ultisnips#540]
I have `let g:UltiSnipsExpandTrigger="<Tab>"`. On a particular system, whenever I press Tab (doesn't matter if I've typed a snippet or not), I get the following error:

```
An error occured. This is either a bug in UltiSnips or a bug in a
snippet definition. If you think this is a bug, please report it to
https://github.com/SirVer/ultisnips/issues/new.

Following is the full stack trace:
Traceback (most recent call last):
  File "<censored>/.vim/bundle/ultisnips/pythonx/UltiSnips/snippet_manager.py", line 57, in wrapper
    return func(self, *args, **kwds)
  File "<censored>/.vim/bundle/ultisnips/pythonx/UltiSnips/snippet_manager.py", line 150, in expand_or_jump
    rv = self._jump()
  File "<censored>/.vim/bundle/ultisnips/pythonx/UltiSnips/snippet_manager.py", line 441, in _jump
    with _vim.toggle_opt('ve', 'onemore'):
  File "/usr/lib64/python2.6/contextlib.py", line 16, in __enter__
    return self.gen.next()
  File "<censored>/.vim/bundle/ultisnips/pythonx/UltiSnips/_vim.py", line 79, in toggle_opt
    command('set {}={}'.format(name, new_value))
ValueError: zero length field name in format
```

It seems like 972305725f5796774e0ac0ed30a47dc1ef5bc1ab introduced the error, prior commits don't have this problem.

This happens on a computer that someone else administers, so I'm not quite sure how Python was installed. `python --version` indicates that Python 2.7.7 is installed; however, `/usr/lib64/python2.7` doesn't exist (there's only `python2.6` in `/usr/lib64`).

I can't reproduce this error on my other computers with Python 2.7.6 (Mac) and Python 3.4.3.

**Comment by SirVer** [comment]
This SO http://stackoverflow.com/questions/5446964/valueerror-zero-length-field-name-in-format-error-in-python-3-0-3-1-3-2 leads me to believe that this is an issue with you using python 2.6. You can verify inside of vim: `:py import sys;print(sys.version)`. 

Python 2.6 is not supported by UltiSnips. You'll need to update to 2.7.

**Comment by hanschen** [comment]
I see, thanks for the fast answer! I'm indeed using Python 2.6 in Vim. I should've looked into it more - thought the problem would be more complicated than that.

I've created a pull request (#541) in case you want to fix this (I completely understand if you don't since 2.6 isn't supported anymore). Ultisnips seems to work fine with Python 2.6 except for this little issue.

**Comment by SirVer** [comment]
Thanks for the patch.

I am not opposed to patches for 2.6, I just have no means of guaranteeing that UltiSnips works with 2.6 anymore, i.e. no testing environment. This means there could be subtle bugs lurking that are not immediately apparent. Therefore it is not officially supported. Also, 2.6 was released in October 2008 - so it is heavily outdated :).

I merged your patch. Thanks for the contribution!
